CODING_RULES.md

```diff
@@ -11,90 +11,156 @@ to be completed
 
 - use getters/setters instead of `@property`
 
-### regarding the exceptions
-
-
-- use exceptions defined in **fedbiomed.common.exceptions**
-
-- callee side: then detecting a python exception in a fedbiomed layer :
-
-  - print a logger.*() message
-
-    - could be logger.critical -> stop the software, cannot continue
-      (ex: FedbiomedEnvironError)
-    - or logger.error -> software may continue (ex: MessageError)
-
-  - raise the exception as a fedbiomed exception
-
-
-- caller/intermediate side: trap the exceptions:
-
-  - separate the FedbiomedError and other exceptions
-
-```
-try:
-
-    something()
-
-except SysError as e:
-    logger.XXX()
-    raise OneOfFedbiomedError()
-```
-
-  - example of top level function (eg: Experiment())
-
-```
-try:
-
-    something()
-
-except FedbiomedError as e:
-    etc...
-
-except Exception as e:   <=== objective: minimize the number of type we arrive here !
-    # place to do a backtrace
-    # extra message to the end user to post this backtrace to the support team
-    etc...
-```
-
-  - except of the top level program, it is **forbidden** to trap all exceptions (with ```except:``` or ```except Exception```)
-
-
-- the **try:** block is as small as possible
-
-- force to read the documentation
-
+### exceptions handling
+
+- on the node: in general, node should not stop because of exceptions that occur during its operation
```

Comment:
Hi @mvesin, 
I think it is better to specify that node should not stop while executing requests received from researcher rather than "its operation". For other operations, usually CLI operations, node can stop with an exception.

Reply:
Thank you, pushed that modification